packages/@coorpacademy-components/src/organism/brand-dashboard/dashboard-popin.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {getOr} from 'lodash/fp';
+import stopPropagation from '../../util/bubbling';
+import Provider from '../../atom/provider';
+import Button from '../../atom/button';
+import style from './dashboard-popin.css';
+
+const DashboardPopin = (props, context) => {
+  const {skin} = context;
+
+  const {ctaLabel, ctaOnClick, header, closeOnClick, content} = props;
+
+  function createMarkup() {
+    return {__html: content};
+  }
+
+  const primary = getOr('#00B0FF', ['common', 'primary'], skin);
+
+  return (
+    <div className={style.default} onClick={closeOnClick}>
+      <div className={style.popin} onClick={stopPropagation}>
```

Comment:
```suggestion
      <div className={style.popin} >
```
il faut creer un handler qui fait un `stopPopagation` et le call à `closeOnClick`